src/MSBuild/XMake.cs

```diff
@@ -3181,18 +3186,19 @@ private static ILogger[] ProcessLoggingSwitches
             string[][] groupedFileLoggerParameters,
             out List<DistributedLoggerRecord> distributedLoggerRecords,
             out LoggerVerbosity verbosity,
+            out LoggerVerbosity originalVerbosity,
             int cpuCount,
             out ProfilerLogger profilerLogger,
             out bool enableProfiler
         )
         {
             // if verbosity level is not specified, use the default
-            verbosity = LoggerVerbosity.Normal;
+            originalVerbosity = verbosity = LoggerVerbosity.Normal;
```

Comment:
nit:
I'd prefer for these to be on separate lines like:
verbosity = LoggerVerbosity.Normal;
originalVerbosity = verbosity;

Reply:
### thanks for the comment, I've resolved in the latest commit